src/multivariate.jl

```diff
@@ -33,9 +119,10 @@ end
 Distributions.params(d::TuringDiagMvNormal) = (d.m, d.σ)
 Distributions.dim(d::TuringDiagMvNormal) = length(d.m)
 Base.length(d::TuringDiagMvNormal) = length(d.m)
-Base.size(d::TuringDiagMvNormal) = (length(d), )
-function Distributions.rand(rng::Random.AbstractRNG, d::TuringDiagMvNormal, n::Int)
-    return d.m .+ d.σ .* randn(rng, length(d), n)
+Base.size(d::TuringDiagMvNormal) = (length(d), length(d))
```

Comment:
This changes the correct implementation to an incorrect one.
```suggestion
Base.size(d::TuringDiagMvNormal) = (length(d),)
```